modules/nsq/src/it/resources/enrich/enrich-nsq.hocon

```diff
@@ -0,0 +1,52 @@
+{
+  "input": {
+    "topic": ${INPUT_TOPIC}
+    "channel": "EnrichNsqChannel"
+    "lookupHost": ${LOOKUP_HOST}
+    "lookupPort": ${LOOKUP_PORT}
+  }
+
+  "output": {
+    "good": {
+      "topic": ${GOOD_OUTPUT_TOPIC}
+      "nsqdHost": ${NSQD_HOST}
+      "nsqdPort": ${NSQD_PORT}
+    }
+
+    "bad": {
+      "topic": ${BAD_OUTPUT_TOPIC}
+      "nsqdHost": ${NSQD_HOST}
+      "nsqdPort": ${NSQD_PORT}
+    }
+  }
+
+  "monitoring": {
+
+    "sentry": {
+      "dsn": ""
+    }
+
+    # Optional, configure how metrics are reported
+    "metrics": {
+
+      # Optional. Log to stdout using Slf4j
+      "stdout": {
+        "period": "10 seconds"
+
+        # Optional, override the default metric prefix
+        # "prefix": "snowplow.enrich."
+      }
+
+      # Optional. Send KCL and KPL metrics to Cloudwatch
+      "cloudwatch": false
```

Comment:
```suggestion
```

It's already [in the defaults](https://github.com/snowplow/enrich/blob/enrich-nsq/modules/nsq/src/main/resources/application.conf#L44-L48).